bin/check-process.rb

```diff
@@ -287,7 +300,7 @@ def run
 
     if config[:metric]
       # #YELLOW
-      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, b| a + b } # rubocop:disable SingleLineBlockParams
+      count = procs.map { |p| p[config[:metric]].to_i }.reduce { |a, e| a + e }
```

Comment:
I don't understand this change or why it would fix rubocop you just renamed the variables...can you expand?